resources/views/contact-tables.blade.php

```diff
@@ -0,0 +1,53 @@
+@extends('components.content-area')
+
+@section('content')
+    @include('components.page-title', ['title' => $page['title']])
+
+    <div class="content">
+        {!! $page['content']['main'] !!}
+    </div>
+
+    <a id="directory-contents"></a>
```

Comment:
Can you check to see if this errors from an automated accessibility check? This tag with no contents likely isn't something we could put live.